src/txn/txn.c

```diff
@@ -1458,6 +1498,8 @@ __wt_txn_stats_update(WT_SESSION_IMPL *session)
 	    stats, txn_commit_queue_len, txn_global->commit_timestampq_len);
 	WT_STAT_SET(session,
 	    stats, txn_read_queue_len, txn_global->read_timestampq_len);
+	WT_STAT_SET(session,
+	    stats, txn_durable_queue_len, txn_global->durable_timestampq_len);
```

Comment:
Again alphabetize this line with its related ones.

Reply:
955effc